bigquery/main.go

```diff
@@ -83,16 +103,49 @@ func (bqf *actualBQFactory) Make(ctx context.Context) (bq, error) {
 	}
 	bqClient, err := bigquery.NewClient(ctx, projectID)
 	if err != nil {
-		return nil, fmt.Errorf("Error intiializing bigquery client: %v", err)
+		return nil, fmt.Errorf("Error intializing bigquery client: %v", err)
 	}
 	newClient := &actualBQ{client: bqClient}
 	return newClient, nil
 }
 
+// GetImageSize calculates the total compressed size of an image by summing it's individual layers
+func GetImageSize(layers []v1.Layer) (*big.Rat, error) {
```

Comment:
Nit: I don't think that you need to export this function. Usually, prefer to keep things unexported as it reduces the dependency surface.